ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/LinearRaoModeller.java

```diff
@@ -36,9 +40,22 @@ public void buildProblem() {
 
     public void updateProblem(LinearRaoData linearRaoData) {
         this.linearRaoData = linearRaoData;
+        fillerList.forEach(filler -> filler.update(linearRaoProblem, linearRaoData));
     }
 
-    public void solve() {
-        //todo
+    public RaoComputationResult solve() {
+        Enum solverResultStatus = linearRaoProblem.solve();
+        RaoComputationResult raoComputationResult;
+        String solverResultStatusString = solverResultStatus.name();
+        if (solverResultStatusString.equals("OPTIMAL")) {
+            RaoComputationResult.Status status = RaoComputationResult.Status.SUCCESS;
+            raoComputationResult = new RaoComputationResult(status);
+            postProcessorList.forEach(postProcessor -> postProcessor.process(linearRaoProblem, linearRaoData, raoComputationResult));
+        } else {
+            RaoComputationResult.Status status = RaoComputationResult.Status.FAILURE;
+            raoComputationResult = new RaoComputationResult(status);
+            LOGGER.warn("Linear rao computation failed: MPSolver status is {}", solverResultStatusString);
+        }
+        return raoComputationResult;
```

Comment:
Might be better to put this method in the LinearRaoProblem and and a getter on the linearRaoProblem here.

Reply:
This function needs the post processors and the data to run so it would not really fit in the LinearRaoProblem.